src/styles/theme.scss

```diff
@@ -10,6 +10,11 @@ $white: #ffffff;
 $black: #000;
 $error: #fb5a5a;
 $black80: #333333;
+$lightBlue: #e8e5fb;
+$darkPurple: #110040;
+$blue: #2a4bff;
+$indigo20T: #{$primary-blue}33;
+$blue40T: #{$blue}66;
```

Comment:
What is the extra 66? Does hex support alpha now? I'm guessing the `T` stands for transparent?

Reply:
Yep, that's the alpha with hex. T is for transparent.